### DISCUSSION TITLE
Question about syllabus

### DISCUSSION THREAD

**Discussion by wpa23** [cooperrc/computational-mechanics#80] [Q&A]
I had a quick question about a few lines in our syllabus. In the section under 'Grading', it lists the breakdown of homework, projects and participation, and then says this:

`Participation: The participation grade will be self-assigned. At week 2 you will submit your current grade and modes of participation, then during the final week submit an updated final grade.`

I was just curious about what this phrase meant and whether or not we have to submit something else for discussion this week.

**Comment by cooperrc** [comment]
Whoops, it should say:

> Participation: The participation grade will be self-assigned. At week ~2~ **7** you will submit your current grade and modes of participation, then during the final week submit an updated final grade.

Its supposed to be a mid-semester self assessment. 

I will post a Google form fo you to fill out in week 7. Thanks for checking in!!
